### ISSUE TITLE
Using {{reflist}} breaks comment parsing

### ISSUE THREAD

**Issue by yuvipanda** [kjschiroo/WikiChatter#19]
See https://en.wikipedia.org/wiki/Wikipedia:Teahouse/Questions/Archive_12#cite_orders

**Comment by kjschiroo** [comment]
By break you mean it incorrectly parses, right? I just want to confirm you aren't having it through an error. Are you referring to `===References===`? Is this auto-generated or common practice? Right now it breaks there because so far as our model of comments is concerned that is a new subsection and it doesn't expect comments to continue across section boundaries. However, if this is a common thing people do I will need to adjust that model.

**Comment by yuvipanda** [comment]
Yup, I think adding a <references> tag automatically outputs a subheading. Let me confirm.

**Comment by yuvipanda** [comment]
Aaah, so <refernces /> itself does not but https://en.wikipedia.org/wiki/Template:Reflist looks like it does.

**Comment by yuvipanda** [comment]
And yes, it does not parse correctly (no errors)

**Comment by kjschiroo** [comment]
@yuvipanda I'm still having trouble seeing how `===References===` is being generated. I've tried using `{{reflist}}` in my sandbox and I'm not getting a header generated in the wikitext. What do I need to do to get this to happen?
